reference/supervisor.md

```diff
@@ -81,6 +81,7 @@ scored.
 
 WbNodeRef wb_supervisor_node_get_from_def(const char *def);
 WbNodeRef wb_supervisor_node_get_from_id(int id);
+const char *wb_supervisor_node_get_def(WbNodeRef node);
```

Comment:
I don't agree.

It is important to keep semantic groups.
So if you say that you want to group all the functions that returns a node, then getId and getDef have to be moved at the end after the other function like getParentNode, getRoot that are returning nodes as well.

Otherwise you have to group the functions related to def name, to the id, etc. mixing sematic groups is bad and makes it difficult to read.